charts/cloud-pricing-api/values.yaml

```
@@ -77,6 +77,8 @@ ingress:
       paths:
         - # -- Path for host
           path: /
+          # -- Path type for this specific host path. https://kubernetes.io/docs/concepts/services-networking/ingress/#path-types
+          pathType: implementationSpecific
```

```suggestion
          pathType: ImplementationSpecific
```
If this needs to be capitalized